config/jobs/etcd/etcd-periodics.yaml

```diff
@@ -90,11 +90,11 @@ periodics:
         VERBOSE=1 GOOS=linux GOARCH=amd64 CPU=8 EXPECT_DEBUG=true GO_TEST_FLAGS=${GO_TEST_FLAGS} RESULTS_DIR=/tmp/results make test-robustness
       resources:
         requests:
-          cpu: "10"
-          memory: "10Gi"
+          cpu: "8"
```

Comment:
on some of our other clusters (not the EKS cluster) our machines are 8 cores so about 7 is as much as you can request, however on all of the clusters 1:1 core:mem is quite low, something like 1:2 is a starting point.

Reply:
thanks Ben for the insight!